client/my-sites/plans/main.jsx

```diff
@@ -36,7 +36,7 @@ const Plans = React.createClass( {
 	getDefaultProps() {
 		return {
 			intervalType: 'yearly',
-			displayJetpackPlans: true
+			displayJetpackPlans: false
```

Comment:
Note: keep in mind that if `displayJetpackPlans` is set to `true`, Jetpack plans will show instead of wpcom ones.

> This is just a conceptual point, but I think this should be defaulted to true. I think the default case should be "show jetpack plans", and that the exception should be to not show them.

I think the opposite: on wp.com, majority of our customers purchase the wp.com plans, not Jetpack ones. If, for some reason, we show Jetpack plans to all the wp.com customers, that would be much worse than showing wp.com plans to a much lower number of Jetpack users/customers (at least currently).

/cc @rralian